### ISSUE TITLE
Warning messages during Upgrade in AIX

### ISSUE THREAD

**Issue by pro-akim** [wazuh/wazuh#26818]
| Version | Revision | Production/Development | Component | Install type |
|:--:|:--:|:--:|:--:|:--:|
|  v4.10.0-alpha3  |  1  | Development   | Agent AIX  | Packages | 

## Description

During the tests performed at https://github.com/wazuh/wazuh/issues/26798, the test AIX Upgrade has displayed the following message

```
# rpm -U wazuh-agent-4.10.0-1.aix6.1.ppc.rpm
warning: /var/ossec/etc/shared/rootkit_trojans.txt created as /var/ossec/etc/shared/rootkit_trojans.txt.rpmnew
rmdir of /var/ossec/tmp/src/init failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates/config/generic/localfile-logs failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates/config/generic failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates/config failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates failed: No such file or directory


```

Seems to be related to: https://github.com/wazuh/wazuh/issues/26817

### Environment

| Node | Vagrant/AWS | Image | CPU | RAM (GB) | Disk size (GB) | Packages/Repository/Sources branch |
|:--:|:--:|:--:|:--:|:--:|:--:|---|
| Wazuh AMI | AWS AMI | AmazonLinux | 4 | 8 | 100 | pre-release |
| Agent AIX | cloud.siteox.com VM | AIX | 2 | 4 | 50 | pre-release |

### Steps to reproduce

1. Deploy a Wazuh AMI
2. Request to DevOps a AIX machine in cloud.siteox.com
3. Install wazuh-agent 4.9.2
4. Upgrade wazuh-agent to v4.10.0-alpha3

### Current result

```
# rpm -U wazuh-agent-4.10.0-1.aix6.1.ppc.rpm
warning: /var/ossec/etc/shared/rootkit_trojans.txt created as /var/ossec/etc/shared/rootkit_trojans.txt.rpmnew
rmdir of /var/ossec/tmp/src/init failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates/config/generic/localfile-logs failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates/config/generic failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates/config failed: No such file or directory
rmdir of /var/ossec/tmp/etc/templates failed: No such file or directory
```

### Expected result

No error messages should be seen.

**Comment by vikman90** [comment]
### Comment

This issue is a known behavior related to the generation of the `rpmsave` files during the uninstallation process. 

The messages reported are not errors but informational messages. They occur because the directories are not fully removed when **certain configuration files or logs are preserved as `rpmsave`**. 

Although the same RPM `SPEC` file is used for both CentOS and AIX, these messages are only observed on AIX due to the **older version of the RPM engine** it uses. The CentOS RPM engine handles this situation differently, which explains why no such messages appear on CentOS.

This behavior is not exclusive to the new 4.10.0 version and has been present in previous versions as well. Therefore, **it is not considered a blocker** for this release.

**Comment by vikman90** [comment]
The reported messages during the upgrade process are mostly **informational** and not indicative of functional errors. These logs are related to the behavior of the **older RPM engine used on AIX**.  

It is possible that these messages have already been addressed with this PR for version 4.10.0:
- https://github.com/wazuh/wazuh/pull/26263)

However, since this issue occurs during the uninstallation of the older package, any fixes would only take effect during **upgrades from 4.10.0** to later versions.  

Given that these issues are **cosmetic and might already be resolved**, we will proceed to close this issue. If similar messages are observed in future versions, we can reopen and investigate further.
